### PR TITLE
liblqr: add test and man installation

### PR DESCRIPTION
**Pull request by equal-l2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/liblqr.rb [hunk 1]
```diff
@@ -33,7 +33,31 @@ class Liblqr < Formula
 
   def install
     system "./configure", "--disable-dependency-tracking",
```
Comment (SMillerDev) [inline Formula/liblqr.rb:35]:
```suggestion
    system "./configure", *std_configure_args,
```

File: Formula/liblqr.rb [hunk 2]
```diff
@@ -33,7 +33,31 @@ class Liblqr < Formula
...
-                          "--prefix=#{prefix}"
+                          "--prefix=#{prefix}",
```
Comment (SMillerDev) [inline Formula/liblqr.rb:36]:
```suggestion
```

**Comment by equal-l2** [comment]
The only failure in test is `dynare`, whose test is known as flaky.

**Review by SMillerDev — DISMISSED** [review]

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**